### PR TITLE
bug/fix : Preamble_text

### PR DESCRIPTION
**Pull request by areyeslo**
# Main
The only valid way to get a valid application_identity is through using now_application_guid field. The current version is causing errors because it is using permit_id to get an application_identity, but that is incorrect.  

- Fixes in backend:
1. The backend logic needs to be updated for permit amendment
2. New SQL update script to filter by now_application_guid.

# Other
- N/A

# How to test
- N/A

# Notes
https://bcmines.atlassian.net/browse/MDS-4150

### CODE REVIEW

File: migrations/sql/V2022.02.22.13.20__update_preambule_text_column_to_existing_draft_permit_amendment_permit.sql [hunk 1]
```diff
@@ -5,7 +5,7 @@ from (select distinct pa.permit_amendment_id, case when atc.application_type_cod
 	  where pa.permit_amendment_status_code = 'DFT' and
       		pa.is_generated_in_core = true and
             pa.preamble_text is NULL and
-            pa.permit_id = nai.permit_id and
+            pa.now_application_guid = nai.now_application_guid and
```
Comment (mmumby) [inline migrations/sql/V2022.02.22.13.20__update_preambule_text_column_to_existing_draft_permit_amendment_permit.sql]: migration files cannot be edited, you must create a new one with a new script.

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js [hunk 2]
```diff
@@ -41,18 +35,24 @@ import {
...
+} from "@/actionCreators/documentActionCreator";
+import CustomPropTypes from "@/customPropTypes";
+import { modalConfig } from "@/components/modalContent/config";
 import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
 import * as Permission from "@/constants/permissions";
 import * as route from "@/constants/routes";
 import NOWTabHeader from "@/components/noticeOfWork/applications/NOWTabHeader";
-import { PERMIT_AMENDMENT_TYPES } from "@common/constants/strings";
+import LinkButton from "@/components/common/buttons/LinkButton";
 import { APPLICATION_PROGRESS_TRACKING } from "@/constants/NOWConditions";
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:56]: why was this removed/added?

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js [hunk 3]
```diff
@@ -315,6 +315,7 @@ export class ProcessPermit extends Component {
       application_last_updated_date: noticeOfWork.last_updated_date
         ? formatDate(noticeOfWork.last_updated_date)
         : formatDate(noticeOfWork.submitted_date),
+        preamble_text: amendment.preamble_text,
     };
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:319]: I thought this was the only update needed on this file.

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js (hunk 2 above)
Comment (areyeslo) [reply services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:56]: This was autoformatted, if you noticed the imports are moved, but are still there. Going to try to remove that change.

File: migrations/sql/V2022.02.22.13.20__update_preambule_text_column_to_existing_draft_permit_amendment_permit.sql (hunk 1 above)
Comment (areyeslo) [reply migrations/sql/V2022.02.22.13.20__update_preambule_text_column_to_existing_draft_permit_amendment_permit.sql]: Wondering about that. Creating a new one.

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js (hunk 3 above)
Comment (areyeslo) [reply services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:319]: it is, this is the only change.

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js (hunk 2 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:56]: phew okay! disregard.

**Review by mmumby — APPROVED** [review]
Thanks for making these fixes! lets get this in and then test again
